llvmlite/tests/test_binding.py

```diff
@@ -783,12 +784,14 @@ def test_set_option(self):
             """
         subprocess.check_call([sys.executable, "-c", code])
 
+    @unittest.skip('Edit this test to test for both llvm14 and llvm15')
```

Comment:
I think this is a leftover from prior to when you had the test passing.
```suggestion
    @unittest.skip('Edit this test to test for both llvm14 and llvm15')
```